README.md

```diff
@@ -1,12 +1,38 @@
 Science Plots
 =============
 
+<p align="left">
```

Comment:
why not keep simple MD?

Reply:
I think all MD tables render a header, even if left blank.
https://stackoverflow.com/questions/17536216/create-a-table-without-a-header-in-markdown

Thanks for the review.

Reply:
sure but MD is simple on purpose so not sure if adding HTML is the right way...

Reply:
HTML is well known IMO. In any case, in the future it can be changed.